README.md

```diff
@@ -1,2 +1,178 @@
-# public-api
-Provides public API interfaces, descriptions and examples
+# Classtime public API
+
+Classtime server works on gRCP-based API. In this repository under [the services directory](https://github.com/gopollock/public-api/tree/main/services), you would be able to find all interface definitions and protobuf messages. However, we also support plain [JSON requests](https://github.com/gopollock/public-api/tree/main#json-requests) based on protobuf messages.
```

Comment:
Some links might not work at this point. They are pointing to a main branch, which doesn't have the required file or folders yet.

Replacing `.../tree/main` with `.../tree/guidance` would help to heal a link.